.github/workflows/deploy-environment.yml

```diff
@@ -138,9 +138,12 @@ jobs:
     needs: [build-image, deploy-db-migrate-service]
     strategy:
       matrix:
-        service_type: [ 'console', 'worker' ]
+        service_type: ['console', 'worker', 'web']
 
     steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
No that matters, but I would also add `if: matrix.service_type == 'web'` as this action is only necessary for web 😄

Reply:
I had wondered if I should do that — done now